### ISSUE TITLE
Null-safe string contains not converted correctly

### ISSUE THREAD

**Issue by chriseldredge** [chriseldredge/Lucene.Net.Linq#48]
Original report: https://github.com/OctopusDeploy/Issues/issues/189
Also reported at: https://github.com/themotleyfool/NuGet.Lucene/issues/10

Queries that do String.Contains on a null safe expression are not converted correctly.

Example where clause:

```
docs.Where(d => (d.Name != null ? d.Name.ToLower() : "").Contains("foo"))
```

**Comment by chriseldredge** [comment]
Fixed in 0e7decbb223cb2c88aa52e8c687f5e08960291b3 by removing redundant null-safety expressions before translating `Contains` method call to a Lucene query expression.

**Comment by chriseldredge** [comment]
Released in build 55: https://www.nuget.org/packages/Lucene.Net.Linq/3.2.55
